src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -186,7 +186,10 @@ internal Silo(string name, SiloType siloType, ClusterConfiguration config, ILoca
 
             logger.Info(ErrorCode.SiloGcSetting, "Silo starting with GC settings: ServerGC={0} GCLatencyMode={1}", GCSettings.IsServerGC, Enum.GetName(typeof(GCLatencyMode), GCSettings.LatencyMode));
             if (!GCSettings.IsServerGC || !GCSettings.LatencyMode.Equals(GCLatencyMode.Batch))
+            {
                 logger.Warn(ErrorCode.SiloGcWarning, "Note: Silo not running with ServerGC turned on or with GCLatencyMode.Batch enabled - recommend checking app config : <configuration>-<runtime>-<gcServer enabled=\"true\"> and <configuration>-<runtime>-<gcConcurrent enabled=\"false\"/>");
+                logger.Warn(ErrorCode.SiloGcWarning, "Note: ServerGC only kicks in on multi-core systems (settings enabling ServerGC have no effect on single-core machines).");
+            }
```

Comment:
The second warning is intentional - the previous warning is already very long, and I feared the note about single-core limitations would get lost in the noise (yes, there's wrapping-around in VS, but other log visualization tools do not necessarily wrap-around).
Also, I thought the alternative - placing `\n\r`s in the warning text to be ugly (is that considered acceptable? - we could do that, I guess).